stable/ambassador/CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 This file documents all notable changes to Ambassador Helm Chart. The release
 numbering uses [semantic versioning](http://semver.org).
 
+## v5.0.0
```

Comment:
Can we add a note in the Upgrading section in the Readme as well ( so it shows up on helm hub ) https://github.com/helm/charts/blob/master/stable/ambassador/README.md#upgrading

Reply:
Nvm :) Missed it